### PR TITLE
Fixes #1990

### LINKED ISSUE #1990: Segmentation fault after consecutive AllChem.AddHs, AllChem.RemoveHs, AllChem.EmbedMolecule
`rdkit.__version__ = '2017.09.2'`
running the code below causes Segmentation Fault
```
from rdkit import Chem
from rdkit.Chem import AllChem
smiles='CCCCCCCCCCCCCCCC(=O)O[C@@H]1CC(C)=C(/C=C/C(C)=C/C=C/C(C)=C/C=C\\C=C(C)\\C=C\\C=C(C)\\C=C\\C2=C(C)C[C@@H](OC(=O)CCCCCCCCCCCCCCC)CC2(C)C)C(C)(C)C1'
mol = Chem.MolFromSmiles(smiles)
mol = AllChem.AddHs(mol)
mol = AllChem.RemoveHs(mol) # Without RemoveHs there is no segmentation fault
AllChem.EmbedMolecule(mol) # the same for AllChem.EmbedMultipleConfs
```

### PR DESCRIPTION
**Pull request by greglandrum**
makes sure we don't destroy a double bond's StereoAtoms just because a bond connected to one of its atoms is removed

### CODE REVIEW

File: Code/GraphMol/molopstest.cpp [hunk 1]
```diff
@@ -7466,6 +7466,41 @@ void testGithub1928() {
...
+                       << std::endl;
+  {
+    std::unique_ptr<RWMol> mol(SmilesToMol("F/C=C/F"));
+    TEST_ASSERT(mol);
+    MolOps::addHs(*mol);
+    MolOps::removeHs(*mol);
+    TEST_ASSERT(mol->getNumAtoms() == 4);
+    TEST_ASSERT(mol->getBondWithIdx(1)->getStereoAtoms().size() == 2);
+  }
+  {  // make sure that stereo is removed when it comes from Hs:
```
Comment (bp-kelley) [inline Code/GraphMol/molopstest.cpp]: Shouldn't this be "stereo is NOT removed"?

Comment (greglandrum) [reply Code/GraphMol/molopstest.cpp]: ? I’m not sure what you’re asking

Comment (bp-kelley) [reply Code/GraphMol/molopstest.cpp]: The PR is for retaining stereo, the comment is stating we ensure stereo is removed, but looking at the examples stereo is retained, so I'm obviously missing something or just confused.

Comment (bp-kelley) [reply Code/GraphMol/molopstest.cpp]: Unless you mean, remove the H atoms from the list of stereo atoms...

Comment (greglandrum) [reply Code/GraphMol/molopstest.cpp]: Ah. I was being stupid. Sorry.
Yes, the comment is misleading. I will fix it.

**Pull request merged**